src/components/Atoms.tsx

```diff
@@ -1,55 +1,23 @@
 import { LinearGradient, LinearGradientProps } from 'expo-linear-gradient'
-import { View, ViewProps, TouchableOpacity, TouchableOpacityProps, ViewStyle } from 'react-native'
+import { Box, Pressable, IPressableProps, IBoxProps, StyledProps, Theme } from 'native-base'
+import { ViewStyle } from 'react-native'
 
-import { useTheme } from '~hooks'
-import { generateStyledSystem, SpacingValue, StyledProps } from '~utils'
-
-export type BoxProps = StyledProps & ViewProps
-export type TouchableProps = StyledProps & TouchableOpacityProps
 export type GradientProps = StyledProps & LinearGradientProps
```

Comment:
I think this will not work for gradient 🤔 

Or you should change the implementation of gradient. Imo you can remove StyledProps for now here
```suggestion
export type GradientProps = LinearGradientProps
```

Reply:
Good catch, definitely it shouldn't be here